crates/language-server/src/functionality/item_info.rs

```diff
@@ -0,0 +1,77 @@
+use hir::{
+    hir_def::{Attr, ItemKind},
+    span::LazySpan,
+    HirDb, SpannedHirDb,
+};
+
+pub fn get_item_docstring(item: &ItemKind, hir_db: &dyn HirDb) -> Option<String> {
+    let docstring = match item {
+        ItemKind::Func(func) => func.attributes(hir_db).data(hir_db).clone(),
+        ItemKind::Mod(mod_) => mod_.attributes(hir_db).data(hir_db).clone(),
+        ItemKind::Struct(struct_) => struct_.attributes(hir_db).data(hir_db).clone(),
+        ItemKind::Enum(enum_) => enum_.attributes(hir_db).data(hir_db).clone(),
+        ItemKind::TypeAlias(type_alias) => type_alias.attributes(hir_db).data(hir_db).clone(),
+        ItemKind::Trait(trait_) => trait_.attributes(hir_db).data(hir_db).clone(),
+        ItemKind::Impl(impl_) => impl_.attributes(hir_db).data(hir_db).clone(),
+        // ItemKind::Body(body) => body.attributes(hir_db).data(hir_db).clone(),
+        // ItemKind::Const(const_) => const_.attributes(hir_db).data(hir_db).clone(),
+        // ItemKind::Use(use_) => use_.attributes(hir_db).data(hir_db).clone(),
+        ItemKind::Contract(contract) => contract.attributes(hir_db).data(hir_db).clone(),
+        _ => vec![],
+    }
```

Comment:
```suggestion
    let docstring = match item {
        ItemKind::Func(func) => func.attributes(hir_db).data(hir_db).,
        ItemKind::Mod(mod_) => mod_.attributes(hir_db).data(hir_db),
        ItemKind::Struct(struct_) => struct_.attributes(hir_db).data(hir_db),
        ItemKind::Enum(enum_) => enum_.attributes(hir_db).data(hir_db),
        ItemKind::TypeAlias(type_alias) => type_alias.attributes(hir_db).data(hir_db),
        ItemKind::Trait(trait_) => trait_.attributes(hir_db).data(hir_db),
        ItemKind::Impl(impl_) => impl_.attributes(hir_db).data(hir_db),
        // ItemKind::Body(body) => body.attributes(hir_db).data(hir_db),
        // ItemKind::Const(const_) => const_.attributes(hir_db).data(hir_db),
        // ItemKind::Use(use_) => use_.attributes(hir_db).data(hir_db),
        ItemKind::Contract(contract) => contract.attributes(hir_db).data(hir_db),
        _ => return None,
    }
```

No need to clone attributes.